components/automate-cli/pkg/verifyserver/models/populatewith_test.go

```diff
@@ -0,0 +1,432 @@
+package models
```

Comment:
Can we rename this file as batchcheck_test.go and name of the function as which function we are testing?